source/scripting_v3/GTA/Entities/Peds/TaskInvoker.cs

```diff
@@ -157,6 +157,184 @@ public void FollowToOffsetFromEntity(Entity target, Vector3 offset, float moveme
 			Function.Call(Hash.TASK_FOLLOW_TO_OFFSET_OF_ENTITY, _ped.Handle, target.Handle, offset.X, offset.Y, offset.Z, movementSpeed, timeout, distanceToFollow, persistFollowing);
 		}
 
+		/// <summary>Tells the <see cref="Ped"/> to perform a task when in a <see cref="Vehicle"/> against another <see cref="Vehicle"/>.</summary>
+		/// <param name="vehicle">The <see cref="Vehicle"/> to use to achieve the task.</param>
+		/// <param name="target">The target <see cref="Vehicle"/>.</param>
+		/// <param name="missionType">The vehicle mission type.</param>
+		/// <param name="cruiseSpeed">The cruise speed for the task.</param>
+		/// <param name="drivingFlags">The driving flags for the task.</param>
+		/// <param name="targetReachedDist">The distance at which the AI thinks the target has been reached and the car stops. To pick default value, the parameter can be passed in as <c>-1</c>.</param>
+		/// <param name="straightLineDist">The distance at which the AI switches to heading for the target directly instead of following the nodes. To pick default value, the parameter can be passed in as <c>-1</c>.</param>
+		/// <param name="driveAgainstTraffic">if set to <see langword="true" />, allows the car to drive on the opposite side of the road into incoming traffic.</param>
+		public void GiveVehicleMission(Vehicle vehicle, Vehicle target, VehicleMissionType missionType, float cruiseSpeed, VehicleDrivingFlags drivingFlags, float targetReachedDist, float straightLineDist, bool driveAgainstTraffic = true)
```

Comment:
I’m feeling like `StartVehicleMission` (or just `VehicleMission`) would be better.

Reply:
Yeah, I think `StartVehicleMission` would fit, due to similarity with `StartScenario`.